### DISCUSSION TITLE
Proposal: Automatically add CSS vendor prefixes using autoprefixer.

### DISCUSSION THREAD

**Discussion by loiclaudet** [okp4/community#3] [OKP4 Portal]
Hi team,
@lolottetheclash, @fredericvilcot,
I want to bring your attention about [this feedback](https://github.com/okp4/dataverse-portal/pull/222#discussion_r1200464465) regarding the vendor prefixes in our SCSS files.

To address this issue and ensure better cross-browser compatibility, I would like to suggest the addition of the [autoprefixer](https://www.npmjs.com/package/autoprefixer) plugin in the Vite configuration.
Autoprefixer is a powerful tool that automatically adds necessary vendor prefixes to CSS properties based on the [Can I Use](https://caniuse.com/) database. By integrating it into our build process, we can save valuable development time and ensure a smoother experience for users across different browsers.

I would love to hear your thoughts on this suggestion. Please feel free to share your opinions and any alternative solutions you might have.

Thank you!

**Comment by fredericvilcot** [comment]
@0xLodz 

Looks like a great improvement, even though the lib size seems quite big for the purpose.

What about the usage with `vitejs` bundler? Have you tried to make it work with our environment tools?

**Reply by loiclaudet** [reply]
Hi @fredericvilcot,

Thanks for your feedback. I tested `autoprefixer` with the current Vite.js setup, and it works well with the existing tooling environment.
To provide a clearer picture, I created [a PR](https://github.com/okp4/dataverse-portal/pull/295) that integrates `autoprefixer` into the Vite config. After making some tests on Safari and Firefox, I confirm it properly automates the addition of vendor prefixes.

I understand your concern regarding the size of the package. While the plugin does increase our development bundle size, it does not impact the size of the JavaScript sent to the client. Its usage is during the build process only and it doesn't have an impact on our app's performance.

Don't hesitate if you have any other concern or question. Thank you!

**Comment by fredericvilcot** [comment]
done in #295
